openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -517,3 +521,37 @@ def _validate_email(email):
         raise AccountEmailInvalid(
             u"Email '{email}' format is not valid".format(email=email)
         )
+
+def set_has_profile_image(username, has_profile_image=True):
```

Comment:
I understand and am open to changing this, but not 100% sure that pattern makes sense for this call.   This function won't ever be invoked directly on behalf of the requesting user, but instead by the profile image system after it completes creating or removing the set of profile images.  Also, since the permissions for upload and remove are different, but they both result in internal calls to this function, this function would have to enforce different rules depending on whether the value of `has_profile_image` was True or False.

Let me know if you feel it's important to follow the pattern anyway, and if so I'll implement the change.